Snowflake.Data.Tests/ChunkDownloaderFactoryTest.cs

```diff
@@ -0,0 +1,102 @@
+﻿/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests
+{
+    using NUnit.Framework;
+    using Snowflake.Data.Configuration;
+    using Snowflake.Data.Core;
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+
+    [TestFixture, NonParallelizable]
+    class ChunkDownloaderFactoryTest
+    {
+        bool UseV2ChunkDownloader = SFConfiguration.Instance().UseV2ChunkDownloader;
+        int ChunkDownloaderVersionDefault = SFConfiguration.Instance().GetChunkDownloaderVersion();
+
+        [TearDown]
+        public void AfterTest()
+        {
+            SFConfiguration.Instance().UseV2ChunkDownloader = UseV2ChunkDownloader; // Return to default version
+            SFConfiguration.Instance().ChunkDownloaderVersion = ChunkDownloaderVersionDefault; // Return to default version
+        }
+
+        [Test]
+        [Ignore("ChunkDownloaderFactoryTest")]
+        public void ChunkDownloaderFactoryTestDone()
```

Comment:
What for are those ignored empty tests?

Reply:
I think those ignored tests were added to track the test progress on github actions: https://github.com/snowflakedb/snowflake-connector-net/pull/464/commits/cf1abe2c940b8b62d8eb97a5837cb31059bfb2d1